cmd/chainbridge/main.go

```diff
@@ -23,11 +23,14 @@ var cliFlags = []cli.Flag{
 	KeystorePathFlag,
 }
 
-var accountFlags = []cli.Flag{
-	ImportFlag,
+var listFlags = []cli.Flag{
 	ListFlag,
 }
 
+var importFlags = []cli.Flag{
+	ImportFlag,
+}
+
```

Comment:
These flags should no longer be required. The aim of this PR is to migrate away from flags to sub-commands

Reply:
I've removed listFlags, but for import Flags, I changed it to read in the First arg, but im not sure if that is correct/always the use case.